projects/canopy/src/lib/side-nav/side-nav-bar-link/side-nav-bar-link.directive.ts

```diff
@@ -0,0 +1,24 @@
+import { Directive, EventEmitter, HostBinding, Input, Output } from '@angular/core';
+
+@Directive({
+  selector: '[lgSideBarNavLink]',
+})
+export class LgSideNavBarLinkDirective {
```

Comment:
I lack a bit of context on this but is the directive approach there to allow us to use standard links and attributes? as opposed to having to map them all to a component?

Reply:
Yeah exactly that. Also, as the intention is that it will use the router, we would then have to ship angular router with Canopy. Whereas this allows the consumer to specific the router links and project them into the side nav